kubernetes/cray-opa/templates/policies/spire.yaml

```diff
@@ -110,6 +113,11 @@ data:
             {"method": "POST", "path": `^/apis/capmc/capmc/v1/get_power_cap$`},
             {"method": "POST", "path": `^/apis/capmc/capmc/v1/get_power_cap_capabilities$`},
             {"method": "POST", "path": `^/apis/capmc/capmc/v1/set_power_cap$`},
+            # PCS - Power control - component power transitioning, power status, and power capping.
```

Comment:
Does it really need to be this open?

Reply:
There really isn't anything we're restricting for PCS. The only endpoints it has are replacements for all of the CAPMC endpoints that are already made available by the OPA policy

Reply:
CAPMC has explicit endpoints. The new PCS ones do not. This is in Spire, needs to be very strictly controlled by endpoint, and also needs to have xname filtering support if enabled. The `wlm` identity is already problematic, as 'one node' could impact more than itself.

Reply:
The current OPA policy for CAPMC rules allow:
- Getting and setting power capping
- Getting power state
- Turning nodes on/off/reinit

These are the only things that PCS does. Having the PCS OPA policy this open does not give the node any more power than it already had. It is just that PCS will be replacing CAPMC